src/Microsoft.TestPlatform.CrossPlatEngine/Hosting/DotnetTestHostManager.cs

```diff
@@ -117,6 +118,8 @@ public DotnetTestHostManager()
             this.processHelper = processHelper;
             this.fileHelper = fileHelper;
             this.dotnetHostHelper = dotnetHostHelper;
+
+            (this.testHostLauncher as DefaultTestHostLauncher)?.SetErrorCallBack(this.ErrorReceivedCallback);
```

Comment:
I don't like this, but to break errorCallBack action from TestProcessStartInfo, this seemed one solution.
Another possible solution would have been to make it part of ITestHostLauncher, which then passes it down to IProcessHelper.

If this doesn't look good at all, let me know, will not push this in.